### PR TITLE
Two columns

### PR DESCRIPTION
**Pull request by isaacdurazo**
@iros, @kadamwhite what do you folks think about this layout for desktop mode?:

screencapture-localhost-3000-1477860480462

### CODE REVIEW

**Comment by kadamwhite** [comment]
@isaacdurazo I recognize the inconsistency in this between the sections, but I think I prefer the bar charts breaking over two lines and the text below; this both makes it easier/possible to see all bars of the chart on one screen, for better comparison, and also allows the bar chart Irene added in #18 (image below) to appear between that section and the text (which can be filtered based on both things).

The section on the top I agree with, and I like what you've done with it!

**Comment by isaacdurazo** [comment]
@kadamwhite yeah, I did it to keep things consistent. Here's a screenshot of the section that also has the bar chart with the emojis and the buttons:
screencapture-localhost-3000-1477927466975

Kadam, I think we should either go with one column for everything or this way. What do you think?

Would also love to know what @iros feels about this

**Comment by iros** [comment]
I think that looks fine @isaacdurazo.

**Comment by iros** [comment]
We'll have to rework the bubble chart a little bit to better occupy the space, but we can do that probably.

**Comment by kadamwhite** [comment]
We agreed to merge this in slack, so doing so

File: src/components/EmbeddedAskForm.scss [hunk 1]
```diff
@@ -42,12 +53,3 @@
...
 }
-
-.form-toggle-button {
-  background: #4a90e2;
-  border-radius: 1em;
-  color: white;
-  display: inline-block;
-  padding: 5px 10px;
-  margin: 2em 0;
-}
```
Comment (kadamwhite) [inline src/components/EmbeddedAskForm.scss:53]: @isaacdurazo Why did you delete this, by the way? I'm reinstating it as I fix up #55

Comment (isaacdurazo) [reply src/components/EmbeddedAskForm.scss:53]: @kadamwhite I'm not sure why I did it. Sorry if I messed up something :/
